packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -192,6 +193,29 @@ class WKScriptMessage {
   final Object? body;
 }
 
+/// Encapsulates the standard behaviors to apply to websites.
+///
+/// Wraps [WKPreferences](https://developer.apple.com/documentation/webkit/wkpreferences?language=objc).
+class WKPreferences {
+  /// Constructs a [WKPreferences].
+  WKPreferences();
+
+  // A WKPreferences that is owned by configuration.
+  WKPreferences._fromWebViewConfiguretion(
```

Comment:
Typo: Configuration

Reply:
I'm surprised that Android Studio didn't pick this up as a typo. Good catch. updated!